packages/aws-cdk-lib/aws-stepfunctions/lib/activity.ts

```diff
@@ -222,4 +269,11 @@ export interface IActivity extends IResource {
    * @attribute
    */
   readonly activityName: string;
+
+  /**
+   * The kms key used by the activity
+   *
+   * @attribute
+   */
+  readonly kmsKey: kms.IKey | undefined;
```

Comment:
Same comment as above, i don't think union types is needed here

Reply:
Again will update to `readonly kmsKey?: kms.IKey;`